cmake/Dependencies.cmake

```diff
@@ -99,6 +99,8 @@ if(USE_XPU)
   foreach(flag ${XPU_HOST_CXX_FLAGS})
     add_definitions(${flag})
   endforeach()
+else()
```

Comment:
I think it does not need this line if my PR is ready, right?

Reply:
No `CMake` related code will be executed from `torch-xpu-ops` so we need to handle it here?

Reply:
Restricted to LINUX systems where XCCL is supported. This solves also the failing build on WINDOWS when this CMake command is executed. Tested locally with XPU OFF and this CMake command works so looks like it's WINDOWS related.